PyDSS/dssInstance.py

```diff
@@ -9,34 +9,31 @@
 from PyDSS import SolveMode
 from PyDSS import pyLogger
 from PyDSS import helics_interface as HI
-from PyDSS.utils.dataframe_utils import write_dataframe
 from PyDSS.utils.utils import make_human_readable_size
-
+from PyDSS.ProfileManager.ProfileStore import ProfileManager
 from PyDSS.exceptions import InvalidParameter, InvalidConfiguration
 
 from PyDSS.pyPostprocessor import pyPostprocess
 import PyDSS.pyControllers as pyControllers
 import PyDSS.pyPlots as pyPlots
-
 import numpy as np
-import pandas as pd
 import logging
+import json
 import time
 import os
 
 from bokeh.plotting import curdoc
 from bokeh.layouts import row
 from bokeh.client import push_session
 from opendssdirect.utils import run_command
-import opendssdirect as dss
-
 
 CONTROLLER_PRIORITIES = 3
 
 class OpenDSS:
     def __init__(self, params):
-        self._TempResultList = []
+        import opendssdirect as dss
```

Comment:
This should be moved to the top of the file with the other imports.

Reply:
done